.github/actions/openvino_provider/action.yml

```diff
@@ -0,0 +1,157 @@
+name: 'OpenVINO provider'
+description: 'Provides OpenVINO build artifacts from a requested source'
+inputs:
+  revision:
+    description: 'Version of OpenVINO to provide - latest_available | HEAD | specific commit hash'
+    required: true
+  branch_name:
+    description: 'Branch of OpenVINO to take the revision from if no specific hash was provided. 
+                  Taken from github context by default'
+    required: false
+  ov_artifact_name:
+    description: "Name under which to upload provided OpenVINO build artifacts, set automatically by default"
+    required: false
+  commit_packages_to_provide:
+    description: "Comma-separated list of OpenVINO packages from post-commit to provide ('all' to get everything)"
+    required: false
+  commit_build_key:
+    description: 'public_linux_ubuntu_20_04_release | public_linux_ubuntu_22_04_release | 
+                  public_linux_ubuntu_24_04_release | public_windows_vs2019_release'
+    required: false
+  commit_share_path:
+    description: 'Path to share holding OpenVINO post-commit artifacts'
+    required: false
+    default: '/mount/build-artifacts'
+  nightly_build_key:
+    description: 'centos7_x86_64 | debian9_armhf | rhel8_x86_64 | ubuntu20_arm64 | ubuntu20_x86_64 | ubuntu22_x86_64 | 
+                  ubuntu24_x86_64 | macos_12_6_arm64 | macos_12_6_x86_64 | windows_x86_64'
+    required: false
+  nightly_package_source:
+    description: 'Source URL hosting OpenVINO nightly archives'
+    required: false
+    default: 'https://storage.openvinotoolkit.org/repositories/openvino/packages/nightly/latest.json'
+  nightly_pip_extra_url:
```

Comment:
Do you plan to add pre-release packages? Now they are in the different channel.

Reply:
But I think this should not be a part of this action, we could add those channels in extensions pipeline directly

Reply:
I believe if you directly specify URL pointing to pre-release package instead of these default values, it will work the same way as for nightly. Later, for convenience, I can add separate defaults, if it will be needed